index.js

```diff
@@ -32,4 +32,21 @@ module.exports = options => options ?
 	getPort(options).catch(() => getPort(Object.assign(options, {port: 0}))) :
 	getPort({port: 0});
 
+module.exports.makeRange = (from, to) => {
+	if (!Number.isInteger(from) || !Number.isInteger(to) || from <= 1024 || to <= 1024 || from > 65535 || to > 65535) {
+		throw new TypeError('`from` and `to` port numbers must be integers in range (1023,65535]');
+	}
+
+	if (from > to) {
+		throw new Error('`from` port must be less than or equal to `to` port');
```

Comment:
Maybe we should use `RangeError` instead. If `from` is greater than `to` we're still violating range's limits.  What do you think?

Reply:
done